pipenv/utils.py

```diff
@@ -282,6 +284,14 @@ def get_requirement(dep):
     :returns: :class:`requirements.Requirement` object
     """
     path = None
+    uri = None
+    editable = False
+    dep_link = None
+    # check for editable dep / vcs dep
+    if isinstance(dep, six.string_types) and dep.startswith('-e '):
```

Comment:
Are we ever passing in non-str types here? We do another `startswith` check immediately below that will fail if that's the case.

Reply:
Not intentionally, no. This is a good catch